karma.config.js

```diff
@@ -35,9 +33,9 @@ module.exports = config => {
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO, // possible values: LOG_DISABLE, LOG_ERROR, LOG_WARN, LOG_INFO, LOG_DEBUG
-    autoWatch: true, // if true, Karma captures browsers, runs the tests and exits
+    autoWatch: false, // if true, Karma captures browsers, runs the tests and exits
     browsers: ['ChromeHeadless'],
-    singleRun: false,
+    singleRun: true,
```

Comment:
Nit:
It is more idiomatic to leave `autoWatch: true` and `singleRun: false` by default (or just remove them altogether, since the default is to run in watch mode), because most of the time people will want to run the unit tests in watch mode.

Then, if you want to run the tests once and exit (as we do in the `test` script), you can pass the `--single-run` option.